pkgs/development/python-modules/django-leaflet/default.nix

```
@@ -0,0 +1,32 @@
+{ pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, python3Packages
+}:
+
+buildPythonPackage rec {
+  pname = "django-leaflet";
+  version = "0.29.0";
```

Upstream has limited their support for the [Python releases](https://github.com/makinacorpus/django-leaflet/blob/0.29.0/CHANGES#LL9C1-L9C50).

```suggestion
  version = "0.29.0";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```